api/v1alpha1/virtualmachine_types.go

```diff
@@ -166,6 +167,11 @@ type Probe struct {
 	// Defaults to 10 seconds. Minimum value is 1.
 	// +optional
 	TimeoutSeconds int32 `json:"timeoutSeconds,omitempty"`
+
+	// PeriodSeconds specifics how often (in seconds) to perform the probe.
+	// Defaults to 10 seconds. Minimum value is 1.
+	// +optional
+	PeriodSeconds int32 `json:"periodSeconds,omitempty"`
```

Comment:
Could you sprinkle in some kubebuilder validation markers for default and minimum here and for TimeoutSeconds above. See https://book.kubebuilder.io/reference/markers/crd-validation.html